time stamp
GoalID goal_id
GoalStatus status
